pyproject.toml

```diff
@@ -46,7 +47,14 @@ pytest-django = "^4.7.0"
 pytest-sugar = "^0.9.7"
 selenium = "^4.15.2"
 factory-boy = "^3.3.0"
+djlint = "^1.34.1"
 
 [build-system]
 requires = ["poetry-core>=1.2.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.djlint]
+profile = "django"
+max_line_length = 120
+ignore = "T002,H006,H021,H025,H030,H031"
```

Comment:
Have you checked these rules ? 
They are defined here : https://djlint.com/docs/linter/
We could try without ignoring any rules first and add some later if we find them annoying?

Reply:
I looked them up but I wasn't able to say what e.g. "Inline styles should be avoided." could mean to us, so I trusted the suggested rules by lavli-food-admin. But we can decide in the future what to not-ignore.